pkg/operation/seed/seed.go

```diff
@@ -1843,3 +1862,37 @@ func cleanupOrphanExposureClassHandlerResources(ctx context.Context, c client.Cl
 
 	return nil
 }
+
+// updatePVCIfStorageNotTheSame updates the PVC if passed storage value is not the same as the current one.
+// If there was a successful update it will retuns true as result.
+// Countion!!!. If the passed storage capacity is less than the current one the existing PCV and its PV will be deleted.
```

Comment:
```suggestion
// updatePVCIfStorageNotTheSame updates the PVC if passed storage value is not the same as the current one.
// If there was a successful update it will return true as result.
// Caution: If the passed storage capacity is less than the current one the existing PVC and its PV will be deleted.
```